test/cli/stats-json/stats-json.exp

```diff
@@ -0,0 +1,7 @@
+{
+  "modulesPaths": [
+    "/Users/trysound/host/fastpack/test/cli/stats-json/index.js",
```

Comment:
The specific absolute path should be replaced with something generic, otherwise it'd fail on CI or anyone's laptop

Reply:
Sure. Are there any utils to work with paths in ocaml?

Reply:
We're currently using the "fileutils" package, see examples of using the FilePath module in the Fastpack/*.ml. Also, maybe the simple replacement of the absolute path with some constant would work. See here: https://github.com/fastpack/fastpack/blob/dafe46a30219556d3dfd4fb13c819cc287da08de/bin/fpack_test.ml#L46